modin/pandas/base.py

```diff
@@ -1247,7 +1246,7 @@ def drop(
                         axes[axis] = None
 
         new_query_compiler = self._query_compiler.drop(
-            index=axes["index"], columns=axes["columns"]
+            index=axes["index"], columns=axes["columns"], errors=errors
```

Comment:
Can we also pass the `errors` parameter to `astype` similar to `drop`?

Reply:
We could, but I don't know of a need to do that on the service side yet. In the case of `drop` here, we bypass the usual error checking at the API layer in line 1232 for the typical client query compiler, so we have to pass `errors` down.

Reply:
It turns out there is a need for `errors` on the service side, so I've made the API layer pass `errors` down.